### ISSUE TITLE
Online / Offline Maps Behaviour | Maps review + Tests

### ISSUE THREAD

**Issue by stacylorraineprendeville** [FundacionParaguaya/MentorApp#937]
![Screen Shot 2019-07-22 at 13.56.45.png](https://images.zenhubusercontent.com/5c19ffa9a896da5d83d6a0d0/5d1dd174-9c04-4682-a546-2a1d116e368f)

review of offline/online maps behavior.
If a user cannot view online map and has offline maps - Offline maps choice should be displayed.

other questions
if a user has a stored location in an offline map - what happens if this map is removed?

**Comment by pA1nD** [comment]
Important for the maps component:
- Write tests for the component
- Review functionality and all the different possible states
- There are differences between how it should have been implemented and how it was understood

**Comment by Danman7** [comment]
I'm putting this in all related issues:

After discussion with Stacy, showing her all the map usecases and current scenarios, we descided that the maps' functionality will remain as it is at the moment. There was an issue with wrong coordinates for one of the survey centers which was addding confusion and was fixed in the backend. I'm removing this issue from any sprints and putting it in the icebox, to bedescided if it's going to be worked on.
